data/tribes/initialization/frisians/units.lua

```diff
@@ -1975,7 +1981,9 @@ wl.Descriptions():new_tribe {
             -- TRANSLATORS: Purpose helptext for a Frisian production site: Reindeer Farm
             purpose = pgettext("frisians_building", "Breeds strong reindeer for adding them to the transportation system. Also keeps them for their fur, which is turned into armor and cloth."),
             -- TRANSLATORS: Note helptext for a Frisian production site: Reindeer Farm
-            note = pgettext("frisians_building", "If the supply is steady, the reindeer farm produces one meat after producing three pieces of fur.")
+            note = pgettext("frisians_building", "If the supply is steady, the reindeer farm produces one meat after producing three pieces of fur."),
```

Comment:
i am ok with the suggestion
Afterwards this should be good to go. 
Wow this will be a lot of translation work, but many thanks for all the effort you put into this